src/resources/genomicsdb_import_config.proto

```diff
@@ -81,11 +81,16 @@ message ImportConfiguration {
   optional bool consolidate_tiledb_array_after_load = 20 [default = false];
   optional bool disable_synced_writes = 21 [default = true];
   optional bool ignore_cells_not_in_partition = 22;
+
   optional int64 lb_callset_row_idx = 23 [ default = 0 ];
   optional int64 ub_callset_row_idx = 24;
+
   optional bool enable_shared_posixfs_optimizations = 27 [ default = false ];
   optional bool disable_delta_encode_for_offsets = 28 [ default = false ];
   optional bool disable_delta_encode_for_coords = 29 [ default = false ];
   optional bool enable_bit_shuffle_gt = 30 [ default = false ];
-  optional bool enable_lz4_compression_gt = 31 [ default = false ];    
+  optional bool enable_lz4_compression_gt = 31 [ default = false ];
+  
+  optional string reference_genome = 32;
+  optional string vcf_header_filename = 33;
```

Comment:
I dunno how much we want to support this, but isn't this one of those options that can be a list or a string?

Reply:
Are you talking about line 92? That was already present. I added 94 and 95 btw.

Reply:
@mlathara, I see that the `vcf_header_filename` [can be a part of partition.](https://github.com/GenomicsDB/GenomicsDB/blob/7973ab6803f00fb7a13db5af28b67a6c4e40ddd7/src/main/cpp/src/config/json_config.cc#L577). Actually, I have hardcoded(with a TODO comment) the `vcf_header_filename` in GenomicsDBImportConfig to '-'/stdout for the CI-tests to pass, I will add the full support just for the sake of completion.

Reply:
Can you be more explicit? Should either `reference_genome` or `vcf_header_filename` be a list? Agreed that the `vcf_output_filename` is part of the `Partition` and have hardcoded that in `GenomicsDBImportConfig::read_from_PB` to stdout, will change that to get it from the Partition for the rank.

Reply:
Sorry - yes, I meant `vcf_header_filename` can be a list

Reply:
One question, @mlathara, do we want to have `vcf_header_filename` to be part of `Partition` and as a general option? Basically, both as a list and a string. The specification in the list will take precedence over that in the string.

Reply:
Yeah - moving to `Partition` seems the right way to do it. I suppose the only reason to have different `vcf_header_filename` values for different ranks is to support MPI-type jobs with no shared filesystem (otherwise, the header content could be the same for all Partitions). That said, I retract my comment -- am fine with leaving things the way they are.

Reply:
I am supporting both(list and string) for the moment as the headers could potentially have an extra line something to the effect of saying that it is a slice from chromosomal position `begin` to `end` and maybe even from `lb_row` to `ub_row`. Something to mull about for later... Thanks.